data_tests/duplicate_entries.py

```diff
@@ -32,8 +31,7 @@ def __hash_row(self, row: list[str]) -> str:
         else:
             entries_to_hash = row
 
-        hashed_entries = [sha256(x.encode()).hexdigest() for x in entries_to_hash]
-        return sha256("".join(hashed_entries).encode()).hexdigest()
+        return hash("".join(entries_to_hash))
```

Comment:
The previous approach of joining the hashes instead of hashing the joined strings was to try and distinguish between distinct rows like
```
"a", "", "b"
"a", "b", ""
```

It looks like our [test for this case](https://github.com/openelections/openelections-data-tests/blob/305f3b1e3ef9cf464458b883bc5c01dff145f078/tests/test_data_tests.py#L79-L95) is [failing](https://github.com/openelections/openelections-data-tests/pull/12/checks?check_run_id=3683137852).  Do you think we can revert to the old approach (but using `hash`), or something similar that achieves the desired behavior?

Reply:
Aha - I was wondering what was behind the original approach. That makes sense.

Instead of incurring the cost of `hash()` for every element, could a separator character work to preserve the empty strings?

Reply:
I think that would work practically all the time.  However, there have been occasions where we've encountered strange characters (e.g., NUL) in the files.  That makes me a bit hesitant to use the separator character approach.

I tried a few different approaches and did some crude timing of testing the Texas repo:
```
$ time python3 run_tests.py duplicate_entries <path to repo>
```
1. Original sha256 approach (305f3b1e3ef9cf464458b883bc5c01dff145f078): 2m50s
2. Separator character approach (b76be0fe4dd16d17a666cafd25657bc23951f8c7): 1m40s
3. Hash a tuple of hashes: 1m43s
  ```
  hashed_entries = (hash(x) for x in entries_to_hash)
  return hash(tuple(hashed_entries))
  ```

What do you think of the third option?

Reply:
Seems snappy enough!

Reply:
Great!  One tiny thing...can you change the return type hint for `__hash_row` from `str` to `int`?